### PR TITLE
Additional Changes

### PR DESCRIPTION
**Pull request by canyon289**
- Added CFP and Code of Conduct to Header
- Hid parts of main page
- Added additional info to CFP pages

Other small fixes like title etc

### CODE REVIEW

**Comment by canyon289** [comment]
@sidravi1 @OriolAbril primarily, but for everyone.

I'm going to self merge this one to get it out and available for the committees to see it without needing to clone and run locally, but PLEASE REVIEW IT ANYWAY. Anything I missed or that should be addressed we can do in follow up PRs

File: _config.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: Your Title
+title: PyMCON 2020
 description: > # this means to ignore newlines until "baseurl:"
   Your description
 baseurl: "" # the subpath of your site, e.g. /blog
```
Comment (OriolAbril) [inline _config.yml:4]: Oh, maybe the issues with the links are due to this not being set to `pymcon`.

Comment (sidravi1) [reply _config.yml:4]: The template we are using doesn't work so well with "project" site. Fixing it is annoying and probably not worth the time. Using relative links works though - probably the way to go

File: _includes/2017_data/team_members.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
         <div class="container">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h2 class="section-heading">Team Members</h2>
+                    <h2 class="section-heading">Planning Committee</h2>
```
Comment (OriolAbril) [inline _includes/2017_data/team_members.html:6]: I think I used organizing committee in CoC, we should make sure to use always the same "title"

**Pull request merged**